packages/codemods/src/plugins/default.ts

```diff
@@ -14,6 +14,34 @@ const defaultCodemodPlugin: CodemodPlugin = {
         return newImport;
       },
 
+      buildAttributes({ originalNode, transformedNode, composedNode }) {
+        if (
+          originalNode.value.type === 'VariableDeclaration' ||
+          originalNode.value.type === 'TaggedTemplateExpression'
+        ) {
+          const newDeclaration = j(originalNode).replaceWith(transformedNode).get();
+
+          if (composedNode) {
+            let candidateForInsertion = newDeclaration;
+
+            while (
+              candidateForInsertion.parentPath &&
+              candidateForInsertion.parentPath.name !== 'body'
```

Comment:
```suggestion
              candidateForInsertion.parentPath?.name !== 'body'
```
are you looping to find the first node that has `parentPath === 'body'`?

Reply:
yep, just need to find a place where we can add a new component declaration